numpy/core/src/npymath/npy_math.c.src

```diff
@@ -130,7 +130,7 @@ double npy_atan2(double y, double x)
         return npy_atan(y);
     }
 
-    m = 2 * npy_signbit(x) + npy_signbit(y);
+    m = 2 * (signbit((x)) != 0 ? 1 : 0) + (signbit((y)) != 0 ? 1 : 0);
```

Comment:
Need to use `npy_signbit` here, `signbit` may not be defined. For instance, I believe it was absent from msvcc prior to 2013. Microsoft C doesn't move very quickly.